src/Umbraco.Infrastructure/Services/Implement/DataTypeService.cs

```diff
@@ -31,13 +31,14 @@ public class DataTypeService : RepositoryService, IDataTypeService
         private readonly ILocalizationService _localizationService;
         private readonly IShortStringHelper _shortStringHelper;
         private readonly IJsonSerializer _jsonSerializer;
+        private readonly IEventAggregator _eventAggregator;
```

Comment:
IEventAggregator is not used because we're using scope instead, I'm guessing this is something left over? (I did the exact same thing 😁